.circleci/windows/prepare.bat

```diff
@@ -32,7 +32,3 @@ C:\Python37\python.exe -m pip install wheel
 REM permanently set variables for Bazel build
 SETX BAZEL_SH "C:\Program Files\Git\usr\bin\bash.exe"
 SETX BAZEL_PYTHON C:\Python37\python.exe
-
-REM TODO: https://github.com/vaticle/typedb/issues/6200
-REM temporary workaround around "AsAbsoluteWindowsPath failed: Unix-style paths are unsupported" error
-DEL .bazelrc
```

Comment:
This is no longer relevant, and unintuitive since we expect windows to read the `.bazelrc` too.